packages/Table/src/index.tsx

```diff
@@ -112,12 +114,17 @@ const Table: FunctionComponent<ITableProps> = ({
             : "keyboard_arrow_down"}
         </i>
       )}
-    </Fragment>
+    </div>
   );
 
+  const tableWrapperClassName = `table__wrapper ${
+    stickyScroll ? "table__wrapper--scroll" : ""
+  }`.trim();
+  const tableClassName = `table ${stickyScroll ? "table--scroll" : ""}`.trim();
+
```

Comment:
I think we are using `classnames` package to do this, will be much cleaner (and consistent).